tests/settings.py

```diff
@@ -115,6 +115,8 @@
 EMAIL_PORT = '1025'  # for testing purposes
 LOGIN_REDIRECT_URL = 'admin:index'
 ACCOUNT_LOGOUT_REDIRECT_URL = LOGIN_REDIRECT_URL
+OPENWISP_ORGANIZATON_USER_ADMIN = True #tests will fail without this setting
```

Comment:
good catch, this is something new in openwisp-users, but please respect PEP8, there should be 2 spaces before `#` and one after:
```python
OPENWISP_ORGANIZATON_USER_ADMIN = True  # tests will fail without this setting
```

Reply:
I didn't really catch it, @atb00ker  suggested :33
I'll take care of the spaces.

Reply:
@shashwat1002 this is still not solved! :(